Docs currency conversion event logs update (#826)
meshcloud/meshcloud-docs: docs/meshstack.billing-configuration.md

<!--END_DOCUSAURUS_CODE_TABS-->

> In order to simplify multi-cloud governance, the best practice is to have single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers (e.g. AWS in USD) chargeback statements will be converted to a single currency. Please be aware that by now we are supporting converison of different currencies such as USD, CNY, TWD, CAD, INR to **Euro** only.

Reviewer: ```suggestion
In order to simplify multi-cloud governance, the best practice is to have a single chargeback currency. meshStack simplifies the chargeback process with automatic currency conversion capability, namely regardless of the original currency from cloud providers (e.g. AWS in USD) chargeback statements will be converted to a single currency. Please be aware that by now we are supporting converison of different currencies such as USD, CNY, TWD, CAD, INR to **Euro** only.
```

You have two options to enable currency conversion for chargebacks and usage reports:

1. **Default converter** fetches rates from [frankfurter.app](https://www.frankfurter.app/), which publishes exchange rates from the European Central Bank. To enable the default converter, simply activate the toggle in the Financial Tab on the Settings page

> 2. If your organization has its own rates, you can utilize **Own converter**, which sends rates to meshStack via API. Simply turn off the toggle button in the Financial Tab of the Settings page and reach out to our support team support@meshcloud.io. They will support you in connecting your own converter with meshStack.

Reviewer: ```suggestion
2. If your organization has its own rates, you can utilize your **Own converter**, which sends rates to meshStack via API. Simply turn off the toggle button in the Financial Tab of the Settings page and reach out to our support team support@meshcloud.io. We will support you in connecting your own converter to meshStack.
```

> For chargebacks, we use the daily exchange rate from the day your statement is finalized, regardless of whether you're using your own converter or the default one. For example,  your chargeback finalization day is March 31st, and the USD to Euro rate on that day is 0.93, then all March chargebacks in USD will be converted and finilized at this rate.

Reviewer: ```suggestion
For chargebacking, we use the daily exchange rate from the day your statement is finalized, regardless of whether you're using your own converter or the default one. For example,  your chargeback finalization day is March 31st, and the USD to Euro rate on that day is 0.93, then all March chargeback statements in USD will be converted and finalized at this rate.
```
I think this might also changed with the changed finalization of the chargeback process. better check

> > Please be aware that activating the default converter or adding your own will only impact future data. Past finalized chargebacks and usage reports won't be retroactively converted to the single currency. So, if you want currency conversion for the current month, make sure to connect or activate it before finalizing chargebacks for that month

Reviewer: ```suggestion
> Please be aware that activating the default converter or adding your own will only impact future data. Past finalized chargeback statements and usage reports won't be retroactively converted to the single currency. So, if you want to enable currency conversion for the current month, make sure to connect or activate it before finalizing chargebacks for that month.
```
This might also change

> If your meshStack is hosted in a private environment and has permission to make external requests, you can activate the default converter using the toggle button on the Settings page, as mentioned earlier. If external requests are restricted, you have the option to enable meshStack to communicate with the Frankfurter API through Firewall rules. Alternatively, if no conversion is desired for chargebacks, you can choose not to apply any conversion by simply turning off togle button in the Financial Tab of the Settings page.

Reviewer: ```suggestion
If your meshStack is hosted in a private environment and has permissions to make external requests, you can activate the default converter using the toggle button on the Settings page, as mentioned earlier. If external requests are restricted, you have the option to enable meshStack to communicate with the Frankfurter API through Firewall rules. Alternatively, if no conversion is desired for chargebacks, you can choose not to apply any conversion by simply turning off toggle button in the Financial Tab of the Settings page.
```
I think this is the first time you call it Frankfurt API which might be confusing to the reader

### Multi-Currency

> If you decide not to use currency conversion and prefer to have multi-currency in chargebacks, simply disable the toggle on the Financial Tab in the Settings page.

Reviewer: ```suggestion
If you decide not to use currency conversion and prefer to have multiple currencies in chargeback statements, simply disable the toggle on the Financial Tab in the Settings page.
```

Reviewer: I think we should further mention that this might lead to broken dashboards

The following product functionalitities do currently not support multi-currency scenarios:

> - **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items are in EUR. This can create confusion if the underlying Tenant Usage Report uses a different or multiple currencies.

Reviewer: Either Tenant Usage Report or tenant usage report. I am for all lower case

> - **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items are in EUR. This can create confusion if the underlying Tenant Usage Report uses a different or multiple currencies.

Reviewer: ```suggestion
- **Detailed Tenant Usage Report**: Detailed tenant usage reports (available for OpenShift, OpenStack, Cloud Foundry and OSB Services) offer additional detail on a Tenant Usage Report. However, the net amount aggregation assumes all line items are in EUR. This can create confusion if the underlying Tenant Usage Report uses different or multiple currencies.
```
